extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -293,9 +293,8 @@ export class AmpLiveList extends AMP.BaseElement {
 
     if (hasNewInsert) {
       promise = promise.then(() => {
-        this.getVsync().mutate(() => {
-          // Should scroll into view be toggleable
-          this.viewport_./*OK*/scrollIntoView(this.element);
+        return this.getVsync().mutatePromise(() => {
+          this.viewport_.animateScrollIntoView(this.element);
```

Comment:
it doesnt return a promise so the return isn't all that helpful i think. (it returns an AnimationPlayer) instance.

@dvoytenko i don't think i need to be in mutate context right? If i remember correctly `Animation.animate` already uses vsync.